.github/workflows/release_scripts/whitesource_vulnurability_checker.py

```diff
@@ -0,0 +1,59 @@
+import requests
+import json
+import os
+import boto3
+
+WS_REST_API_URL = 'https://saas.whitesourcesoftware.com/api'
+WS_API_KEY = os.getenv('WS_APIKEY')
+WS_PROJECT_TOKEN = os.getenv('WS_PROJECTTOKEN')
+WS_BLOCKING_VULNERABILITIES = "critical,high"
+
+def determine_any_exclusions():
+    dynamodb_client = boto3.resource('dynamodb')
+
+    whitesource_exclusion_table = dynamodb_client.Table('whitesource-excluded-libraries')
+    whitesource_exclusion_entries = whitesource_exclusion_table.scan()['Items']
+
+    exclusions = set()
+    for exclusion in whitesource_exclusion_entries:
+        exclusions.add(exclusion['library'])
+
+    return exclusions
+
+
+def find_all_high_critical_vulnerabilities_to_resolve(excluded_libraries):
+    ws_payload = {
+        'requestType': 'getProjectVulnerabilityReport',
+        'format': 'json',
+        'userKey': WS_API_KEY,
+        'projectToken': WS_PROJECT_TOKEN
+    }
+    headers = {'content-type': 'application/json'}
+    ws_vulnerability_report_response = requests.request(
+        'post',
+        WS_REST_API_URL,
+        data=json.dumps(ws_payload),
+        headers=headers)
+
+    project_ws_vulnerabilities = json.loads(ws_vulnerability_report_response.text)
```

Comment:
As a suggestion , would ws_vulnerability_report_response.json() work here if the content-type is application/json ?

Reply:
sounds good, replaced it! thanks